pkg/cloud/services/compute/service.go

```diff
@@ -39,6 +39,17 @@ type Service struct {
 	logger            logr.Logger
 }
 
+/*
+ The minimum Nova microversion supported by CAPO
+ 2.53 corresponds to OpenStack Pike
+
+ For the canonical description of Nova microversions, see
+ https://docs.openstack.org/nova/latest/reference/api-microversion-history.html
+
+ CAPO uses server tags, which were added in microversion 2.52.
+*/
+const NovaMinimumMicroversion = "2.53"
```

Comment:
I think microversion is >= ? so 2.52 is good here?

Reply:
I rounded it up to a released version, and Pike is 2.53.